ydb/core/blobstorage/vdisk/skeleton/blobstorage_skeletonfront.cpp

```diff
@@ -1094,6 +1094,17 @@ namespace NKikimr {
                 ev->Record.SetReplicationSecondsRemaining(0);
             }
             ctx.Send(SelfId(), ev.release());
+            TActorId NodeWardenServiceId = MakeBlobStorageNodeWardenID(VCtx->NodeId);
+            ctx.Send(NodeWardenServiceId,
+                     new TEvBlobStorage::TEvControllerUpdateDiskStatus(
+                         SelfVDiskId,
+                         SelfId().NodeId(),
+                         Config->BaseInfo.PDiskId,
+                         Config->BaseInfo.VDiskSlotId,
+                         0,
```

Comment:
you're sure about 0? this will overwrite any valid value during merge; we possibly would want std::optional<> here and skip this field in ctor